Example/PaymentSheet Example/PaymentSheet Example/ExampleSwiftUICustomPaymentFlowCVCRecollection.swift

```diff
@@ -137,6 +144,19 @@ class MyCustomBackendCVCRecollectionModel: ObservableObject {
             })
         task.resume()
     }
+    func intentConfiguration() -> PaymentSheet.IntentConfiguration {
```

Comment:
If we move this functionality (ability to _update_ FlowController w/ new IntentConfiguration) to the playground, can we delete this file?

Reply:
Yep!